src/main/config/simplified_tuning.h

```diff
@@ -23,7 +23,8 @@
 #include "flight/pid.h"
 #include "sensors/gyro.h"
 
-#define SIMPLIFIED_TUNING_MIN 0
+#define SIMPLIFIED_TUNING_PIDS_MIN 0
+#define SIMPLIFIED_TUNING_FILTERS_MIN 10
```

Comment:
```suggestion
#define SIMPLIFIED_TUNING_FILTERS_MIN 50
```

Reply:
If we change to 50 we have to update configurator also.

Reply:
Where is this limit defined in the configurator? I couldn't seem to find it.

Reply:
The range is defined in html on the input element.

Reply:
Ah it's defined in `pid_tuning.html` right? I was only looking in js files.

Reply:
Sorry, my fault. I was looking at an old configurator for some weird reason...